doc/docs/FAQ.md

```diff
@@ -73,10 +73,16 @@ If you are worried about this, then you are probably setting up your calculation
 
 ### How is the source current defined?
 
-The source current in Meep is defined as a [free charge current $\mathcal{J}$ in Maxwell's equations](Introduction.md#maxwells-equations). Meep does not simulate the driving force behind this free charge current, nor does the current have to be placed in a conductor. Specifying a current means that somehow you are shaking a [charge](https://en.wikipedia.org/wiki/Electric_charge) at that point (by whatever means, Meep doesn't care) and you want to know the [resulting fields](#how-does-the-current-amplitude-relate-to-the-resulting-field-amplitude). In a linear system, multiplying $\mathcal{J}$ by 2 results in multiplying the fields by 2.
+The source current in Meep is defined as a [free-charge current $\mathcal{J}$ in Maxwell's equations](Introduction.md#maxwells-equations). Meep does not simulate the driving force behind this free-charge current, nor does the current have to be placed in a conductor. Specifying a current means that somehow you are shaking a [charge](https://en.wikipedia.org/wiki/Electric_charge) at that point (by whatever means, Meep doesn't care) and you want to know the [resulting fields](#how-does-the-current-amplitude-relate-to-the-resulting-field-amplitude). In a linear system, multiplying $\mathcal{J}$ by 2 results in multiplying the fields by 2.
+
+Note that a free-charge current in a polarizable medium such as a dielectric material (including vacuum) creates [bound charges](https://en.wikipedia.org/wiki/Maxwell%27s_equations#Bound_charge_and_current) which in turn can act as a secondary current source. In an FDTD simulation, these bound charges $\rho_b$ (which are equivalent to the divergence of the polarization field $\vec{P}$ via Gauss's law or [electric dipole moment](https://en.wikipedia.org/wiki/Electric_dipole_moment)) may make it seem that the current source never turns off because the fields do not decay away to zero. This is demonstrated in the figure below showing a plot of the electric energy over the entire cell versus simulation time for a point dipole in 3d vacuum surrounded by PML. To ensure that the net dipole moment is zero when the source turns off, the source should be the *integral* of $\mathcal{J}$ rather than just $\mathcal{J}$. This feature is enabled in Meep by setting the `is_integrated` parameter of the [`GaussianSource`](Python_User_Interface.md#gaussiansource) class object to `True` (default is `False`).
```

Comment:
The integral of free current is free polarization (hence free charge), not bound charge.  (Of course, free charges *generate* polarization of the bound charge if you have a polarization, but the *source* of the field is ultimately the free charge density).